src/main.cpp

```diff
@@ -36,6 +36,8 @@
 
 #ifdef WEMOSOLED // display via i2c for WeMOS OLED board
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 4, 5, U8X8_PIN_NONE);
+//#elif ESP32Sboard // display via i2c for ESP32S board
```

Comment:
Maybe @danielbernalb these lines are a unnecessary comments?

Reply:
Yes, is better in other site like the gui